stonesoup/platform/tests/test_platform_base.py

```diff
@@ -649,6 +649,11 @@ def test_range_and_angles_to_other(first_state, second_state, expected_measureme
                                transition_model=None)
 
     range_, azimuth, elevation = platform1.range_and_angles_to_other(platform2)
+    # Allow azimuth of -pi to equal pi, to avoid unnecessary assert failures
+    if azimuth == -np.pi:
+        azimuth = np.pi
+    if expected_measurement[1] == -np.pi:
```

Comment:
Expected measurement is never `pi`, so lines 655-656 are not needed. Could potentially be removed.

Reply:
Expected measurement is currently pi a few times, but not -pi. These lines are pre-emptive defence against someone writing new tests using -pi, since there doesn't seem to be a clear convention as to which to use I'd keep them. Unless pre-emptive defence = confusing clutter...?

Reply:
> Expected measurement is currently pi a few times, but not -pi. These lines are pre-emptive defence against someone writing new tests using -pi, since there doesn't seem to be a clear convention as to which to use I'd keep them. Unless pre-emptive defence = confusing clutter...?

Fair enough: agree with the argument.